nat_gateway/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "private_route_tables" {
```

Comment:
How will this work if I have for example 2 x 3 private subnets (eg proxy and app subnet)?